internal/dependency/manager/manager_test.go

```diff
@@ -0,0 +1,288 @@
+package manager
+
+import (
+	"github.com/g2a-com/klio/internal/context"
+	"github.com/g2a-com/klio/internal/dependency"
+	"github.com/spf13/afero"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+//==========================================================================================
+//::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+//==========================================================================================
+// Take a look at DEP INDEX in utils_test.go
+//==========================================================================================
+
+func TestGetInstalledCommands(t *testing.T) {
+	tests := []struct {
```

Comment:
Test-cases array like this, don't play nicely with VS Code (you loose ability of run/debug single test-case). Split it into separate functions.